src/app.ts

```diff
@@ -1,126 +1,126 @@
-import "reflect-metadata";
-import express from "express";
-import dotenv from "dotenv";
-import path from "path";
-import morgan from "morgan";
-import expressSession from "express-session";
-import cookieParser from "cookie-parser";
-import { RequestAuthentication } from "./util/hs_auth";
-import { Express, Request, Response, NextFunction } from "express";
-import { Connection, createConnections, ConnectionOptions } from "typeorm";
-import { errorHandler, error404Handler } from "./util/errorHandling";
-import { RouterInterface } from "./routes";
-import { TYPES } from "./types";
-import container from "./inversify.config";
+import 'reflect-metadata';
+import dotenv from 'dotenv';
+import path from 'path';
+import morgan from 'morgan';
+import expressSession from 'express-session';
+import cookieParser from 'cookie-parser';
+import { RequestAuthentication } from './util/hs_auth';
+import express, { Express, Request, Response, NextFunction } from 'express';
+import { Connection, createConnections, ConnectionOptions } from 'typeorm';
+import { errorHandler, error404Handler } from './util/errorHandling';
+import { RouterInterface } from './routes';
+import { TYPES } from './types';
+import container from './inversify.config';
 
 // Load environment variables from .env file
-dotenv.config({ path: ".env" });
```

Comment:
We should probably move this to the top of the file to avoid the same problem we had in hs_applications where the env variables aren’t being loaded

Reply:
Done